slides/en/index.html

```diff
@@ -407,6 +475,11 @@ <h2>Forms &amp; Pages</h2>
             </ul>
 
             <aside class="notes">
+			  4<br /><br />
+			  Form: return some input from the end user, associated to process (process instantiation or human task)<br />
```

Comment:
Worth to mention that form also trigger the execution from engine perspective.

Reply:
Agreed